### ISSUE TITLE
Document common reports

### ISSUE THREAD

**Issue by boris-moduscreate** [toolsplus/atlassian-marketplace-sheets-connector#2]
I am working on building some new reports, however it would be helpful if the repo also contained example reports that I could clone.

**Comment by tbinna** [comment]
Hey @boris-moduscreate, atm we do not have any examples to share but I agree that an example would be useful to get started. We are using this connector to get data into [Looker Studio](https://lookerstudio.google.com/).

**Comment by boris-moduscreate** [comment]
Yeah I am doing the same thing. Just trying to see if there is anything you may be able to share. Possibly as a template? https://developers.google.com/looker-studio/connector/report-templates
